src/scripts/num.js

```diff
@@ -80,6 +80,31 @@ export function percent(number = 0) {
   )
 }
 
+export function percentBig(number = 0) {
+  let formatted = Math.round(number * 100) / 100
+
+  let suffix = ""
+  if (Math.abs(Number(formatted)) >= 1e12) {
+    formatted = Number(formatted) / 1e12
+    suffix = "T"
+  } else if (Math.abs(Number(formatted)) >= 1e9) {
+    formatted = Number(formatted) / 1e9
+    suffix = "B"
+  } else if (Math.abs(Number(formatted)) >= 1e6) {
+    formatted = Number(formatted) / 1e6
+    suffix = "M"
+  } else if (Math.abs(Number(formatted)) >= 1e3) {
+    formatted = Number(formatted) / 1e3
+    suffix = "K"
+  }
+  return (
+    new Intl.NumberFormat(language, {
+      minimumFractionDigits: 0,
+      maximumFractionDigits: 1
+    }).format(formatted) + ` ${suffix} %`

```

Comment:
removing '%' you could use the same logic for all the values.

Reply:
like `setDecimalLength`

Reply:
Yes, this is what I was thinking actually